### DISCUSSION TITLE
Rstudio addon?

### DISCUSSION THREAD

**Discussion by mertgoksel** [marp-team/marp#402] [Ideas]
I would love to use this with Rstudio, given that Rstudio already has markdown support it shouldnt be hard to implement i think.

**Comment by yhatt** [comment]
Marp team has no plans to create a new integration for other IDEs at this moment.

We have an open ecosystem, and already provide tools and libraries enough to create an integration for something ([Marpit](https://marpit.marp.app) framework, [Marp Core](https://github.com/marp-team/marp-core) and [Marp CLI](https://github.com/marp-team/marp-cli)). We hope our community create a new integration for other tools by using them.
